projects/core/src/user/store/selectors/user-replenishment-orders.selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { createSelector, MemoizedSelector } from '@ngrx/store';
+import { ReplenishmentOrderList } from '../../../model/replenishment-order.model';
+import { StateUtils } from '../../../state/utils/index';
+import { StateWithUser, UserState } from '../user-state';
+import { LoaderState } from '../../../state/utils/loader/loader-state';
+import { getUserState } from './feature.selector';
+
```

Comment:
no unit test

Reply:
should have two other selectors for usage if needed for fail and loading

Reply:
will follow up